core/payment/src/dao/agreement.rs

```diff
@@ -94,7 +92,6 @@ pub fn increase_amount_scheduled(
     amount: &BigDecimal,
     conn: &ConnType,
 ) -> DbResult<()> {
-    assert!(amount > &BigDecimal::zero().into()); // TODO: Remove when payment service is production-ready.
```

Comment:
only called on requestor.

Added protection from this: https://github.com/golemfactory/yagna/pull/1195/files#diff-e7feca8bba1ad54d3dcd4f1405598227535fbf8078840b5f584cc44eaa1f1169R431